pkg/cloud/cloud.go

```diff
@@ -732,7 +740,7 @@ func (c *cloud) waitForSnapshotCreate(ctx context.Context, snapshotID *string) e
 			case "completed":
 				return true, nil
 			case "pending":
-				return false, nil
+				return true, nil
```

Comment:
Since we are always returning true here, do we still need this `waitForSnapshotCreate` when creating snapshot? Can we just check the snapshot state returned from the `CreateSnapshotWithContext`?

Reply:
Yes, this is rather paranoid -- I only wanted to be sure `DescribeSnapshotsWithContext` can find the newly created snapshot.